src/main/scala/com/cognite/sdk/scala/v1/resources/dataModelInstances.scala

```diff
@@ -46,7 +46,7 @@ class DataModelInstances[F[_]](
   )(implicit F: Async[F]): F[ItemsWithCursor[DataModelInstanceQueryResponse]] = {
     implicit val printer: Printer = Printer.noSpaces.copy(dropNullValues = true)
 
-    dataModels.retrieveByExternalIds(Seq(inputQuery.modelExternalId)).flatMap { dm =>
+    dataModels.retrieveByExternalIds(Seq(inputQuery.modelExternalId), true).flatMap { dm =>
```

Comment:
Can we make this into a parameter in `query` function with default value true? Since this is an SDK, it could be better to allow both ways.

Reply:
as discuss the call to `retrieveByExternalIds`  will be remove when moving to the new api anyway so I just leave true here 😁